DESCRIPTION

```diff
@@ -27,11 +27,18 @@ Imports:
     AzureRMR,
     AzureStor,
     cli,
+    data.table,
     DBI,
+    dplyr,
     duckdb,
     EpiNow2 (>= 1.4.0),
-    rlang
+    jsonlite,
+    rlang,
+    rstan,
+    tidybayes
+Additional_repositories:
+    https://mc-stan.org/r-packages/
```

Comment:
there is a new repo for stan packages now

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/41/commits/4e01a51c5df01a97cab8d2d37a3cd19e16813c6c